libraries/Adapters/Microsoft.Bot.Builder.Adapters.Slack/SlackHelper.cs

```diff
@@ -227,7 +227,8 @@ public static Activity EventToActivity(EventRequest eventRequest, SlackClientWra
                     Id = innerEvent.User ?? innerEvent.BotId ?? eventRequest.TeamId
                 },
                 ChannelData = eventRequest,
-                Type = ActivityTypes.Event
+                Type = ActivityTypes.Event,
+                ServiceUrl = "https://slack.botframework.com/"
```

Comment:
We could put this value in a constant to be able to reuse it in the other methods if necessary.

Reply:
moved to a constant, ty